sqliterepo/repository.c

```diff
@@ -1347,6 +1348,11 @@ sqlx_repository_use_base(sqlx_repository_t *repo, const struct sqlx_name_s *n,
 			err = _base_lazy_recover(repo, n, status);
 		}
 
+		if (status == STEP_MASTER && notify_master) {
+			GRID_WARN("Possible double master");
```

Comment:
I suggest adding the name of the base to the message.
```suggestion
			GRID_WARN("Possible double master for [%s][%s]", n->base, n->type);
```